src/components/tipRecords/FormatDate.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <span>{{formatDate}}</span>
```

Comment:
Can you change it to something like this
```
  <time datetime="2008-02-14 20:00" title="2008-02-14 20:00">{{ formatDate }}</time>
```

Reply:
Title is very helpful to get the exact time, because now it will will be hidden after 1 day